docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -166,7 +166,7 @@ It is possible to provide a GET parameter named `since` where the value need to
 ```
 ex: 2016-08-09 or 2016-08-09 10:00:00 or 2016-08-09T10:00:00-05:00
 ```
-The format must follow php Supported Date and Time Formats for Datetime objects. See http://php.net/manual/en/datetime.formats.php  
+We recommand using a format that include timezone.
```

Comment:
"recommend", not "recommand"

This rephrasing also lost any mention of what's valid. Maybe, to avoid referencing PHP and mention the timezone:

"The format must follow ISO 8601. We recommend using a variation which includes timezone."

(Though I'm not sure if the "2016-08-09 10:00:00" example is valid ISO 8601).

Reply:
also should be "includes"

Reply:
There is 3 option examples just above. I am going to fix typos.